src/assisted_test_infra/test_infra/controllers/assisted_installer_infra_controller.py

```diff
@@ -0,0 +1,296 @@
+import logging
+import time
+from enum import Enum
+
+import kubernetes.client
+import kubernetes.config
+import kubernetes.dynamic
+import waiting
+
+
+class AssistedInstallerInfraController:
+    """Manage minikube or kind (TBD) commands / config changes via k8s_client
```

Comment:
```suggestion
    """Manage hub cluster configuration changes via kubernetes commands.
```

Reply:
Rename any occurrence of "minikube" (in PR description, commit, code changes, etc.)